ideascube/tests/helpers.py

```diff
@@ -0,0 +1,37 @@
+from django.apps import apps
+from django.test import TransactionTestCase
+from django.db.migrations.executor import MigrationExecutor
+from django.db import connection
+import pytest
+
+
+@pytest.mark.skipif(
+    pytest.config.getoption("--nomigrations"),
+    reason="skipping due to --nomigrations"
+)
+class MigrationTestCase(TransactionTestCase):
```

Comment:
I don't know how to create py.test fixtures parametrized by the value known at test definition.
Fixtures can be parametrized or use another fixtures but this is defined at fixture creation.

The `migrate_from` and `migrate_to` are relative to test, and so it's not possible to create a generic fixture here. (And I agree, it's a pity :/ )